### DISCUSSION TITLE
Enable redstone machines to work outside of claims.

### DISCUSSION THREAD

**Discussion by dahneta** [GriefPrevention/GriefPrevention#1342] [Q&A]
Currently, on my geyser server, redstone machines only work inside claimed areas. What command or setting do I change to enable redstone machine outside of claims as well? I’ve looked through the config file but can’t see anything that may achieve this. Thanks

**Comment by Jikoo** [comment]
GP doesn't limit redstone outside of claims, only pistons. That's the [PistonMovement config option](https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#pistonmovement).

**Reply by dahneta** [reply]
So, any ideas what is stopping redstone signals? Has to be something to do with GP as redstone works inside claims but doesn’t work outside claims.

**Reply by bigpresh** [reply]
Can you expand on "doesn't work"?  What kind of machines are you talking about?  Do no redstone features work, or just pistons?  If it's just pistons, but other things (redstone lamps, dispensers, etc) work, see the doco which Jikoo linked. 

I've no idea what Geyser does or if it would present events differently to Spigot/Paper.  Do you get the same issue when running e.g. Spigot or Paper?

**Reply by Jikoo** [reply]
I would look to any plugins you have that advertise redstone control or that you know are modifying GP behavior. You can try with only GP installed to demonstrate that it isn't a GP feature you're encountering - GP [does not listen to the BlockRedstoneEvent](https://github.com/TechFortress/GriefPrevention/search?q=BlockRedstoneEvent).
